docs/tutorials/reinforce_reacher_gym_v26.py

```diff
@@ -0,0 +1,249 @@
+"""
+Solving Reaper-v4 using REINFORCE
+=================================
+
+**This tutorial aims to familiarise users with Gymnasium v26’s new
+.step() function.**
+
+   In v21, the type definition of step() is
+   tuple\ ``[ObsType, SupportsFloat, bool, dict[str, Any]`` representing
+   the next observation, the reward from the step, if the episode is
+   done, and additional info from the step. Due to reproducibility
+   issues that will be expanded on in a blog post soon, we have changed
+   the type definition to
+   tuple\ ``[ObsType, SupportsFloat, bool, bool, dict[str, Any]``]
+   adding an extra boolean value. This additional bool corresponds to
+   the older done, which has been changed to terminated and
+   truncated.These changes were introduced in Gym v26 (turned off by
+   default in v25).
+
+In this tutorial, we will be using the REINFORCE algorithm [1] to solve
+the task of *Reacher*. Reacher is a two-jointed robot arm. The goal is
+to move the robot’s end effector (called a fingertip) close to a target
+that is spawned at a random position.
+
+"""
+
+import gymnasium as gym
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.distributions import Categorical
+from torch.distributions.normal import Normal
+
+import pandas as pd
+import seaborn as sns
+import random
+
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+plt.rcParams["figure.figsize"] = (10,5)
+
+import warnings
+warnings.filterwarnings("ignore")
+
+env = gym.make("Reacher-v4")
+
+class Policy_Network(nn.Module):
```

Comment:
Could you add a description to explain what the policy network does, why this architecture, what its input and output is